flexget/plugins/sites/piratebay.py

```diff
@@ -65,20 +62,39 @@ class UrlRewritePirateBay(object):
         ]
     }
 
+    url = None
+
+    def __init__(self):
+        self.set_urls(URL)
+
+    def on_task_start(self, task, config=None):
+        if not isinstance(config, dict):
+            config = {}
+        self.set_urls(config.get('url', URL))
+
+    def set_urls(self, url):
+        url = url.rstrip('/')
+        if self.url != url:
+            self.url = url
+            parsed_url = urlparse(url)
+            self.url_match = re.compile('^%s://(?:torrents\.)?(%s)/.*$' % (re.escape(parsed_url.scheme),
+                                                                           re.escape(parsed_url.netloc)))
```

Comment:
Is this pep8-compliant? I'd expect it to align with the re.escape above.

Reply:
Yes, this is compliant. Alignment is based on the parentheses, which is the python-preferred way using the opening delimiter as anchor.

Reply:
Hmm, when I was looking at this at home it was definitely not compliant. I can see now (at work) that it is. I'll chalk it up to ninja-edits or my monospace font acting up.